source/geometry/FloatTransform3D.ooc

```diff
@@ -253,6 +254,16 @@ FloatTransform3D: cover {
 	createProjection: static func (focalLength: Float) -> This {
 		This new(focalLength, 0, 0, 0, 0, focalLength, 0, 0, 0, 0, focalLength, 1.0f, 0, 0, 0, 0)
 	}
+	createProjection: static func ~flip (focalLength, nearPlane, farPlane: Float, coordinateTransform: IntTransform2D) -> This {
+		if (focalLength > 0.0f) {
+			a := (coordinateTransform a as Float) * focalLength
+			f := -(coordinateTransform e as Float) * focalLength
+			k := (farPlane + nearPlane) / (farPlane - nearPlane)
+			o := 2.0f * farPlane * nearPlane / (farPlane - nearPlane)
+			FloatTransform3D new(a, 0.0f, 0.0f, 0.0f, 0.0f, f, 0.0f, 0.0f, 0.0f, 0.0f, k, -1.0f, 0.0f, 0.0f, o, 0.0f)
```

Comment:
Creating a projection matrix for OpenGL's axis system. Maybe it does not belong here but we already have a projection function above.